script.service.ltvpl/addon.xml

```diff
@@ -0,0 +1,57 @@
+<addon id="script.service.ltvpl" name="Live TV Playlist" provider-name="jmooremcc" version="1.0.0">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0" />
+    <import addon="xbmc.addon" version="17.0.0" />
+    <import addon="xbmc.gui" version="5.14.0" />
+  </requires>
+  <extension library="service.py" point="xbmc.service" />
+  <extension library="resources/lib" point="xbmc.python.module" />
+  <extension library="resources" point="xbmc.python.module" />
+  <extension library="addon.py" point="xbmc.python.script">
+    <provides>executable</provides>
+  </extension>
+  <extension point="kodi.context.item">
+    <menu id="kodi.core.main">
+      <item library="addon.py">
+        <label>30090</label>
+        <visible>Window.IsVisible(tvguide)|Window.IsVisible(tvsearch)</visible>
+      </item>
+    </menu>
+  </extension>
+  <extension debugging="false" effectslowdown="0.75" point="xbmc.gui.skin">
```

Comment:
this is only for skins. you're not a skin.

Reply:
removed